sieve/sieve-lex.l

```diff
@@ -317,7 +317,7 @@ ws              [ \t]+
     /* notify - draft-martin-sieve-notify-01 */
 <INITIAL>denotify             return DENOTIFY;
 <INITIAL>:method              return METHOD;
-<INITIAL>:id                  return ID;
+<INITIAL>:id                  return NID;
 <INITIAL>:low                 return yylval->nval = LOW;
```

Comment:
Perhaps sieved.c shall be changed to print NID instead of ID?

Reply:
No, because the actual parameter is :id.  Support for draft-martin-sieve-notify-01 will probably be stripped out soon.